sdk/java-sdk-spring/src/it/java/com/example/wiring/workflowentities/WorkflowWithRecoverStrategy.java

```diff
@@ -29,7 +29,7 @@
 import java.util.concurrent.CompletableFuture;
 
 import static java.time.Duration.ofSeconds;
-import static kalix.javasdk.workflow.Workflow.RecoverStrategy.maxRetries;
+import static kalix.javasdk.workflow.AbstractWorkflow.RecoverStrategy.maxRetries;
```

Comment:
This could potentially break the build for users, but I think no one is using workflows on prod at this moment.

Reply:
We do have a few people building PoCs with Workflows.

What about moving the RecoveryStategy to an interface that is then implemented by `Workflow` and `ProtoWorkflow`?

Reply:
I'm not sure if I follow you on that, what would that solve?

Reply:
Ok, I'm merging this as it is, I would like to solve it in a separate PR once the scala part is done.